src/traccuracy/matchers/_ctc.py

```diff
@@ -79,15 +79,16 @@ def _match_ctc(self):
             )
             pred_label_to_id = {v: k for k, v in pred_labels.items()}
 
-            overlapping_gt_labels, overlapping_pred_labels = get_labels_with_overlap(
-                gt_frame, pred_frame
-            )
+            (
+                overlapping_gt_labels,
+                overlapping_pred_labels,
+                intersection,
+            ) = get_labels_with_overlap(gt_frame, pred_frame)
+
             for i in range(len(overlapping_gt_labels)):
                 gt_label = overlapping_gt_labels[i]
                 pred_label = overlapping_pred_labels[i]
-                gt_blob_mask = gt_frame == gt_label
-                comp_blob_mask = pred_frame == pred_label
-                if detection_test(gt_blob_mask, comp_blob_mask):
+                if intersection[i] > 0.5:
```

Comment:
```suggestion
# CTC metrics only match comp IDs to a single GT ID if there is majority overlap
                if intersection[i] > 0.5:
```

Since we're not using the `detection_test` function anymore, it would be great to add a comment for the otherwise "magic" 0.5 value, and delete the `detection_test` function entirely.